test/rules/no-static-only-classes/test.ts.lint

```diff
@@ -0,0 +1,60 @@
+export class PascalClass {
+             ~~~~~~~~~~~ [0]
+    public static helper(): void {}
+    private static privateHelper(): boolean {
+        return true;
+    }
+    constructor() {}
+}
+
+export class PascalClass {
+             ~~~~~~~~~~~ [0]
+    public static helper(): void {}
+    private static privateHelper(): boolean {
+        return true;
+    }
+}
+
+export class ClassA implements IInterfacey {
+    public static helper(): void {}
+    private static privateHelper(): boolean {
+        return true;
+    }
+    constructor() {}
+}
+
+export class ClassB extends BaseClass {
+    public static helper(): void {}
+    private static privateHelper(): boolean {
+        return true;
+    }
+    constructor() {}
+}
+
+export class PascalClass {
+    public static helper(): void {}
+    private static privateHelper(): boolean {
+        return true;
+    }
+    constructor() { this.that = what; }
+}
+
+export class PascalClass {
+    public static helper(): void {}
+    private static privateHelper(): boolean {
+        return true;
+    }
+    public index = 0;
+    constructor() {}
+}
+
+export class PascalClass {
+    public static helper(): void {}
+    private Helper(): boolean {
+        return true;
+    }
+    constructor() { this.that = what; }
+}
```

Comment:
I'd like a test for shorthand properties as well, e.g.:

```typescript
export class PascalClass {
    public static helper(): void {}
    constructor(private that: string) {}
}
```

This should not raise a warning because the `private that` declares a non-static property.